README.md

```diff
@@ -156,9 +172,19 @@ To run the test accession file at `tests/test-data/acc-url.csv`, run:
 sourmash scripts urlsketch tests/test-data/acc-url.csv -o test-urlsketch.zip -f out_fastas -k --failed test.failed.csv -p dna,k=21,k=31,scaled=1000,abund -p protein,k=10,scaled=100,abund -r 1
 ```
 
+### Usage Considerations
```

Comment:
duplicated text?

Reply:
one for each command. But i can bump up a level and just include it once.

Reply:
done in [b583fb9](https://github.com/sourmash-bio/sourmash_plugin_directsketch/pull/102/commits/b583fb9a04c6df61b0ae7903fd2a12adb7c9bf7f)